packages/core/test/new-api/getParameter.ts

```diff
@@ -1,93 +1,128 @@
 import { assert } from "chai";
 
 import { defineModule } from "../../src/new-api/define-module";
-import { NamedContractDeploymentFutureImplementation } from "../../src/new-api/internal/module";
+import { ModuleParameterRuntimeValueImplementation } from "../../src/new-api/internal/module";
 import { ModuleConstructor } from "../../src/new-api/internal/module-builder";
+import { ModuleParameterType } from "../../src/new-api/types/module";
+
+import { assertInstanceOf } from "./helpers";
 
 describe("getParameter", () => {
```

Comment:
Replaced these tests with a pretty simple version. They just test that it returns the right object, and that it supports all the types.